locales/en/app.ftl

```diff
@@ -562,3 +562,97 @@ rec-section-subhead-no-pw = Though passwords weren’t exposed in this breach, t
 
 # Button
 see-additional-recs = See Additional Recommendations
+
+# This string contains nested markup that becomes a link later in the code.
+# Please do not modify or remove "<a>" and "</a>".
+resolve-top-notification = { $affectedEmail } appeared in this breach. <a>What to do next</a>
+
+marking-this-subhead = Marking this breach as resolved
+
+# This string contains nested markup that is later used to style the text inside of it.
+# Please do not modify or remove "<span>" and "</span>".
+marking-this-body = 
+  <span>Once you’ve taken the steps you can to address this breach</span>,
+  you can mark it as resolved. You can still access details about the breach 
+  from your dashboard at any time.
+
+# Button
+mark-as-resolve = Mark as Resolved
+
+# Button
+marked-as-resolved = Marked as Resolved
+
+# Button
+undo = Undo
+
+confirmation-1-subhead = Nice! You’ve just resolved your first breach.
+confirmation-1-body = Keep up the momentum. Check your dashboard to see if there’s more to do.
+confirmation-2-subhead = Take that, hackers!
+confirmation-2-body = You’re taking important steps towards protecting your online accounts.
+confirmation-3-subhead = Another one down. Nice work!
+
+# This string contains nested markup that becomes a link later in the code.
+# Please do not modify or remove "<a>" and "</a>".
+confirmation-3-body = Is your new password unique, strong, and hard to guess? <a>Find out</a>
+
+generic-confirmation-1-subhead = This breach has been marked as resolved
+generic-confirmation-status =
+  { $numRemainingBreaches ->
+      [one] There is { $numRemainingBreaches } unresolved breach remaining.
+     *[other] There are { $numRemainingBreaches } unresolved breaches remaining. 
+  }
+generic-confirmation-complete = You’ve marked all known breaches as resolved.
+
+# Link
+return-to-breach-details = Return to breach details
+
+# Link
+go-to-dashboard = Go to Dashboard
+
+# This string appears above a breach resolution progress bar and indicates
+# the percentage of breaches a user has resolved. For instance, "27% complete".
+progress-percent-complete = { $percentComplete } complete
+
+# This string appears in the purple callouts at the top of the user dashboard and shows
+# the total number of breaches a user has resolved. For instance, "5 Resolved".
+num-resolved = { $numResolved } Resolved
+
+progress-intro-subhead = New in { -product-name }: Mark breaches as resolved
+progress-intro-message =
+  After reviewing the details about a breach and taking steps to protect 
+  your personal info, you can mark breaches as resolved.
+progress-status = { $numResolvedBreaches} out of { $numTotalBreaches } breaches marked as resolved
```

Comment:
Same here. It doesn't affect English, but this should be a plural string based on `$numTotalBreaches`